### PR TITLE
Add missing swedish translations !1718

### PR DESCRIPTION
**Pull request by didair**
## What does this PR do?
Adds missing Swedish translations for e-mails.

## Test Plan
Set appwrite locale to Swedish by setting the X-Appwrite-Locale to sv.
Generate emails using e.g. POST /v1/account/recovery [Account recovery (docs)](https://appwrite.io/docs/client/account?sdk=web-default#accountCreateRecovery)

## Related PRs and Issues
#1718 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yes

### CODE REVIEW

**Comment by didair** [comment]
> 
> 
> Could you please complete the PR description 🙂

Updated! I hope this is OK

File: app/config/locale/translations/sv.json [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
+    "emails.verification.subject": "Verifiera konto",
+    "emails.verification.hello": "Hej {{name}}",
+    "emails.verification.body": "Klicka på denna länk för att verifiera din email",
+    "emails.verification.footer": "Om du inte bad om att verifiera den här e-postadressen kan du ignorera detta mail.",
+    "emails.verification.thanks": "Tack",
+    "emails.verification.signature": "{{project}} teamet",
+    "emails.magicSession.subject": "Logga in",
+    "emails.magicSession.hello": "Hej,",
+    "emails.magicSession.body": "Klicka på denna länk för att logga in.",
+		"emails.magicSession.footer": "Om du inte bad om att logga in med denna e-postadress kan du ignorera detta mail.",
```
Comment (christyjacob4) [inline app/config/locale/translations/sv.json:15]: Can we please fix this space ?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

**Comment by christyjacob4** [comment]
@didair  Looks good. Just left a minor comment

File: app/config/locale/translations/sv.json (hunk 1 above)
Comment (didair) [reply app/config/locale/translations/sv.json]: Done

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**